addons/binding/org.openhab.binding.jeelink/ESH-INF/i18n/jeelink_de.properties

```diff
@@ -0,0 +1,59 @@
+thing-type.lacrosse.description = Ein mit dem JeeLink USB Empf\u00e4nger verbundener Lacrosse Temperatur-Sensor.
+thing-type.ec3k.label = ec3k
+thing-type.ec3k.description = Ein mit dem JeeLink USB Empf\u00e4nger verbundenes EC3000 Energiekosten-Messger\u00e4t.
+
+# parameters
+parameter.portname.label = Port Name
+parameter.portname.description = Der Name des Ports, an den der Jeelink angeschlossen ist.
+parameter.sketchname.label = Sketch Name
+parameter.sketchname.description = Der Name des geladenen Sketches.
+parameter.initcommands.label = Init Kommandos
+parameter.initcommands.description = Optionale Initialisierungs-Kommandos (durch Semikolon getrennt) die nach erfolgreicher Verbindung an den Jeelink geschickt werden, z.B. "0a v" um die LED auszuschalten.
+parameter.sensorid.label = Sensor ID
+parameter.sensorid.description = Die Sensor ID dieses Sensors.
+parameter.updateinterval.label = Update Intervall
+parameter.updateinterval.description = Das Update Intervall in Sekunden (0 setzt den Sensor in den 'live' Modus, d.h. jeder Wert wird an OpenHAB weitergereicht).
+parameter.sensortimeout.label = Sensor Timeout
+parameter.sensortimeout.description = Die Anzahl von Sekunden, nach deren Ausbleiben von Messwerten der Sensor Status auf OFFLINE gesetzt wird.
+parameter.buffersize.label = Buffer Gr\u00f6\u00dfe
+parameter.buffersize.description = Die Anzahl von Messwertden, die zur Berechnung des Durchschnitts benutzt werden (0 schaltet die Durchschnittsberechnung ab).
+parameter.mintemp.label = Minimal g\u00fcltige Temperatur
+parameter.mintemp.description = Die niedrigste vom Sensor gelesene, als g\u00fcltig erachtete Temperatur. Alle niedrigeren Werte werden ignoriert.
+parameter.maxtemp.label = Maximal g\u00fcltige Temperatur
+parameter.maxtemp.description = Die h\u00f6chste vom Sensor gelesene, als g\u00fcltig erachtete Temperatur. Alle h\u00f6heren Werte werden ignoriert.
+
+# channel types
+channel-type.current-watt-channel.label = Momentaner Verbrauch
+channel-type.current-watt-channel.description = Der momentane Verbrauch des angeschlossenen Ger\u00e4tes in W.
+channel-type.max-watt-channel.label = H\u00f6chster Verbrauch
+channel-type.max-watt-channel.description = Der h\u00f6chste Verbrauch des angeschlossenen Ger\u00e4tes in W.
+channel-type.consumption-total-channel.label = Absoluter Verbrauch
+channel-type.consumption-total-channel.description = Der absdolute Verbrauch des angeschlossenen Ger\u00e4tes in W.
+channel-type.appliance-time-channel.label = Einschaltzeit des Ger\u00e4tes
+channel-type.appliance-time-channel.description = Die Anzahl der Sekunden w\u00e4hrend deren das angeschlossene Ger\u00e4t eingeschaltet war.
+channel-type.sensor-time-channel.label = Einschaltzeit des Sensoren
+channel-type.sensor-time-channel.description = Die Anzahl der Sekunden w\u00e4hrend deren der Sensor eingeschaltet war.
+channel-type.resets-channel.label = Resets
+channel-type.resets-channel.description = ???
```

Comment:
You do not know it yourself? Then maybe simply leave the description empty or remove the channel altogether.